test/Microsoft.Azure.SignalR.E2ETests/Management/ServiceHubContextE2EFacts.cs

```diff
@@ -310,8 +311,8 @@ internal async Task StopServiceHubContextTest()
                         o.ServiceTransportType = ServiceTransportType.Persistent;
                     })
                     .Build();
-                var serviceHubContext = await serviceManager.CreateHubContextAsync("hub", loggerFactory);
-                var connectionContainer = ((ServiceHubContext)serviceHubContext).ServiceProvider.GetRequiredService<IServiceConnectionContainer>();
+                var serviceHubContext = await serviceManager.CreateHubContextAsync("hub", default);
+                var connectionContainer = ((ServiceHubContext)serviceHubContext).ServiceProvider.GetRequiredService<IServiceConnectionContainer>();//TODO
```

Comment:
remove this `//TODO` comments with little meanings, or add more details into it about what todo

Reply:
This test is for reconnection function and doesn't work anymore, because `connectionContainer` isn't in the service provider of a `ServiceHubContext` and it's not convenient to add it to the service collection per hub. I think ideally this test should do for connection container class. I intend to add a unit test for connection container separately latter.